src/ResourceManager/EventGrid/Commands.EventGrid/help/New-AzureRmEventGridSubscription.md

```diff
@@ -149,8 +149,8 @@ This can be webhook or eventhub
 ```yaml
 Type: String
 Parameter Sets: ResourceGroupNameParameterSet, ResourceIdEventSubscriptionParameterSet, CustomTopicEventSubscriptionParameterSet
-Aliases: 
-Accepted values: webhook, eventhub
+Aliases:
+Accepted values: webhook, eventhub, webhook, eventhub
```

Comment:
remove the repeats here and on line 166

Reply:
@maddieclayton  This text is generated using the commands here: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md 

Is this a bug in the tool? I can manually remove these but it seems an infra/tool issue. 

Also, I regenerated the file but I did not see any changes in the md files. I had to add text to the help to see the md files to be updated. however, the repeated texts was still shown. Also, it had bad format in the md file in which yaml keywords were not in new lines.

I will have to reformat this manually too.

Am i missing something? are these known issues?